doc/code_style/mom.h

```diff
@@ -18,13 +18,13 @@
 
 // Includes are sorted alphabetically,
 // global includes first, then system inclusions:
-#include <cstddef>
-#include <cstdint>
-
 #include "../magic/unicorn.h"
 #include "rofl.h"
 #include "stuff.h"
 
+#include <cstddef>
+#include <cstdint>
```

Comment:
the include order should be preserved

Reply:
`IncludeBlocks: 'Preserve'`, what we saw is `Regroup`